lineapy/graph_reader/artifact_collection.py

```diff
@@ -271,8 +271,12 @@ def _compose_module(
                 f"Duplicated input parameters {module_input_parameters_list} across multiple sessions"
             )
         elif set(module_input_parameters_list) != set(self.input_parameters):
+            missing_parameters = set(self.input_parameters) - set(
+                module_input_parameters_list
+            )
             raise ValueError(
-                f"Detected input parameters {module_input_parameters_list} do not agree with user input {self.input_parameters}"
+                f"Detected input parameters {module_input_parameters_list} do not agree with user input {self.input_parameters}. "
+                + f"The following parameters are missing {missing_parameters}."
```

Comment:
missing might be the wrong word here? its not that the parameter is required and not passed. its that the session did not have reference to the input parameter that user provided.

Reply:
Changed

Reply:
"The following variables do not have references in any session code"